apps/settings/js/panels/bluetooth/panel.js

```diff
@@ -88,8 +88,7 @@ define(function(require) {
             actionMenuDialog: panel.querySelector('.paired-device-option'),
             connectOption: panel.querySelector('.connect-option'),
             disconnectOption: panel.querySelector('.disconnect-option'),
-            unpairOption: panel.querySelector('.unpair-option'),
-            cancelOption: panel.querySelector('.cancel-option')
```

Comment:
I can't find a call site for `cancelOption`, or a place where it's still accessed in code. Can you point to a line number?